packages/proto/lib/coresdk.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('./root').coresdk;
+module.exports.temporal = require('./root').temporal;
+module.exports.google = require('./root').google;
```

Comment:
Re-added & edited `coresdk.js` & `.d.ts` in order to get the build working (we have code importing from `@temporalio/proto/lib/coresdk`). I could either maintain backcompat for anyone importing from `/lib/*` by also adding a similar `temporal.js,.d.ts`, or I could remove this file and update our imports.

Reply:
Let's just get rid of this, it's not worth keeping, no reason for it anymore.